File: app/src/main/java/com/feriantes4dawin/feriavirtualmovil/dependencies/FeriaVirtualDBProvider.java
package com.feriantes4dawin.feriavirtualmovil.dependencies;

import android.content.Context;
import com.feriantes4dawin.feriavirtualmovil.data.db.FeriaVirtualDatabase;
import com.feriantes4dawin.feriavirtualmovil.data.db.FeriaVirtualDatabase;

public class FeriaVirtualDBProvider {

    public static FeriaVirtualDatabase provideDB(Context c){

        return FeriaVirtualDatabase.getInstance(c);

    }


}

File: app/src/main/java/com/feriantes4dawin/feriavirtual/dependencies/FeriaVirtualDBProvider.kt
package com.feriantes4dawin.feriavirtual.dependencies

import android.content.Context
import com.feriantes4dawin.feriavirtual.data.db.FeriaVirtualDatabase

object FeriaVirtualDBProvider {

    fun provideDB(c:Context):FeriaVirtualDatabase{

        return FeriaVirtualDatabase.getInstance(c)

    }


}